modules/geolocation/geolocation.py

```diff
@@ -129,3 +129,38 @@ def calculate_pixel_to_geo_mapping(self):
 
         # Return matrix product of mappedGeoMatrix and sourcePixelMatrixInverse
         return (mappedGeoMatrix.dot(sourcePixelMatrixInverse))
+
+    def map_location_from_pixel(self, pixels, coordinates):
```

Comment:
`pixels` is not a good name. It's the transform matrix that is acting on the input pixels (which is currently called `coordinates`).

Reply:
Fixed! Changed 'pixels' to 'transformation_matrix' and 'coordinates' to 'pixels'.